ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoData.java

```diff
@@ -59,7 +60,7 @@
      * @param cracVariantId:     Existing variant of the CRAC on which RaoData will be based
      * @param loopflowCountries: countries for which we wish to check loopflows
      */
-    public RaoData(Network network, Crac crac, State optimizedState, Set<State> perimeter, ReferenceProgram referenceProgram, GlskProvider glskProvider, String cracVariantId, Set<Country> loopflowCountries) {
+    public RaoData(Network network, Crac crac, State optimizedState, Set<State> perimeter, ReferenceProgram referenceProgram, GlskProvider glskProvider, String cracVariantId, Set<Country> loopflowCountries, double loopFlowAcceptableAugmentation) {
```

Comment:
I don't like the idea of adding a parameter in the constructor of RaoData. 

It makes this class even more complicated than it already is :p

I know that there is "raoParameters.getLoopflowCountries()", but this is questionable as many other raoParameters are used within the RaoData methods even though there are not in the constuctor.

Reply:
perfect for the change, thank you :cake: